### DISCUSSION TITLE
predicted concentration vs observed concentration

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#777] [Questions & Problems] [answered]
Dear to all,
After  parameters  identified, we can see the  scatter plot of  predicted concentration vs observed concentration, and whether the scatter plot is after parameter identification or before parameter identification?
![image](https://user-images.githubusercontent.com/63133525/91116625-53f61680-e6bf-11ea-93a8-d81f94fdac4b.png)

**Comment by msevestre** [comment, marked as answer]
Please refer to the documentation
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#analyses


On Mon, Aug 24, 2020 at 10:43 PM Wenjing1018 <notifications@github.com>
wrote:

>
>
> Dear to all,
>
>
> After parameters identified, we can see the scatter plot of predicted
> concentration vs observed concentration, and whether the scatter plot is
> after parameter identification or before parameter identification?
>
>
> [image: image]
> <https://user-images.githubusercontent.com/63133525/91116625-53f61680-e6bf-11ea-93a8-d81f94fdac4b.png>
>
>
>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Open-Systems-Pharmacology/Forum/issues/594>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAH6GVMN4SFZ3SK3UZGR3D3SCMQOBANCNFSM4QKEO5FQ>
> .
>
>
> --
Kind Regards,

Michael


--


*Michael Sevestre*

*Design2Code Inc.*


Email michael@design2code.ca

Website www.design2code.ca

**Comment by Wenjing1018** [comment]
@ msevestre，
I tried to read it a few times, but I still couldn't understand. What I want to know is that this scatter plot corresponding to the fitted parameter value or corresponding to the previous parameter?
Kind Regards,
Wenjing

**Comment by Yuri05** [comment]
All parameter identification plots display **optimization results** (based on fitted parameter values)

<img width="600" src="https://user-images.githubusercontent.com/25061876/91146006-95ce8f00-e6b6-11ea-9bac-90eb0d1210ea.png">

**Comment by Wenjing1018** [comment]
@Yuri05，
Your reply is very helpful for me ,thank you very much!
Kind Regards

**Comment by fdelbene76** [comment]
@Yuri05
How I can get goodness-of-fit plots (e.g., PRED vs OBS, Residuals vs Time, and so on) of my current model prior to the parameter identifications?
Many thanks in advance.

Kind regards,
Francesca

**Comment by StephanSchaller** [comment]
Hi Francesca,

there might be two options:
1. run the simulations i R and create the plots manually in R
2. run the Parameter Identification but reduce the number of iterations in the configuration to zero and plot the (unfitted) results in the parameter identification interface

Hope this helps,
Stephan

**Comment by fdelbene76** [comment]
Hi Stephan,

The second option is a quick and easy solution for me.

Thanks a lot.

Kind regards,
Francesca
